test/unittests/util/test_util.py

```diff
@@ -0,0 +1,227 @@
+from os.path import (abspath, dirname, expanduser, join, normpath, isdir,
+                     exists)
+import shutil
+import tempfile
+from unittest import TestCase, mock
+
+from mycroft import MYCROFT_ROOT_PATH
+from mycroft.util import (resolve_resource_file, curate_cache,
```

Comment:
convert to new directory structure (e.g. `from mycroft.util.audio_util`, etc.).  should these tests be broken up into one file per module being tested?

Reply:
`mycroft.util` should still export all these methods so I think it's better for the test to pull from the module and not the submodules